autoload/wiki/page.vim

```diff
@@ -68,6 +68,14 @@ function! wiki#page#rename(newname) abort "{{{1
     return
   endif
 
+  " The target directory must exist
+  let l:target_dir = fnamemodify (expand (l:newpath), ':h')
+  if !isdirectory (l:target_dir)
+    echom 'wikiError: Cannot rename to "' . l:newpath . '".
```

Comment:
I agree it is already better to give a specific warning. But I think we can ask the user for input on whether or not to create the directory. We can create it with `call mkdir(..., 'p')`. We can use `input()` to ask for input.

Reply:
I also think we should improve the error message in line 84, e.g. `throw 'wiki.vim error: Cannot rename file!`.